client/packages/requisitions/src/RequestRequisition/api/hooks/utils/useIsRemoteAuthorisation.ts

```diff
@@ -0,0 +1,17 @@
+import { RequisitionNodeApprovalStatus } from '@common/types';
+import { useRequest } from '..';
+
+export const useIsRemoteAuthorisation = () => {
+  const { approvalStatus, linkedRequisition } = useRequest.document.fields([
+    'approvalStatus',
+    'linkedRequisition',
+  ]);
+
+  const linkedRequisitionStatus =
+    linkedRequisition?.approvalStatus ?? RequisitionNodeApprovalStatus.None;
+  const usesRemoteAuthorisation =
+    approvalStatus != RequisitionNodeApprovalStatus.None ||
```

Comment:
this is for requestRequisition, i don't think we need to check approvalStatus on requestRequisition (just the linkedRequistiion.approvalStatus)

Reply:
yeah, that works. the request status would only be updated by the linked req status anyway.